static/app/views/settings/organizationDeveloperSettings/index.tsx

```diff
@@ -155,22 +155,32 @@ class OrganizationDeveloperSettings extends AsyncView<Props, State> {
   renderBody() {
     return (
       <div>
-        <SettingsPageHeader title={t('Developer Settings')} />
-        <PanelAlert type="info">
+        <div>
+          <SettingsPageHeader
+            title={t('Developer Settings')}
+            body={t(
+              `Create integrations to interact with Sentry using the REST API and webhooks.`
+            )}
+            action={
+              <Button
+                external
+                href="https://docs.sentry.io/product/integrations/integration-platform/"
+              >
+                {t('View Docs')}
+              </Button>
+            }
+          />
+        </div>
+        <Alert type="info">
           {tct(
-            "We've added new webhooks for comments on issues! Read more in our [link:webhook documentation].",
+            'You can create integrations with webhooks notifying you when a comment on an issue is added or changes.  [link:Learn more].',
```

Comment:
> with webhooks notifying you when
I think this would be clearer as
"You can create integrations that have webhooks that notify you when a comment on an issue is added or changed."